Beep bop, your totally non-human CI bot.

```
$ go test -v
# github.com/SUSE/pistis
# [github.com/SUSE/pistis]
./pistis.go:183:3: github.com/SUSE/pistis.Debug format %s has arg pgpObj of wrong type *github.com/ProtonMail/go-crypto/openpgp.Entity
FAIL    github.com/SUSE/pistis [build failed]
```

--- a/pistis.go
+++ b/pistis.go
@@ -180,7 +180,6 @@ func logic() {
 		Debug("Author %s", commit.Author)
 		Debug("Committer %s", commit.Committer)
 		Debug("Signature %s", commit.PGPSignature)
-		Debug("pgpObj %s %v", pgpObj, pgpObj)
 
 		handleError("Verifying commit", verifyErr)
 		cFp := hex.EncodeToString(pgpObj.PrimaryKey.Fingerprint[:])

